src/utilcode/longfilepathwrappers.cpp

```diff
@@ -1366,12 +1366,26 @@ HRESULT LongFile::NormalizePath(SString & path)
         }
     }
 
-
-    //wcscpy_s always termintes with NULL, so we are saving the character that will be overwriiten
-    WCHAR temp = buffer[prefix.GetCount()];
-    wcscpy_s(buffer, prefix.GetCount() + 1, prefix.GetUnicode());
-    buffer[prefix.GetCount()] = temp;
-    path.CloseBuffer(ret + prefixLen);
+	SString fullpath(SString::Literal, buffer + prefixLen);
+
+	if (fullpath.BeginsWith(UNCPathPrefix) && prefixLen != prefix.GetCount() - 2)
+	{
+		// must initialize before calling SString::Empty()
+		SString::Startup();
```

Comment:
Can we actually get here without already having called `SString::Startup()`? Also, I don't see `SString::Empty()` being called below. Is it called indirectly by another method? Might be a good idea to clarity in the comment.